### DISCUSSION TITLE
Choice for other chat frameworks

### DISCUSSION THREAD

**Discussion by NK-Nikunj** [UIUC-PPL/charm#3644] [General]
Hi everyone,

We were thinking about moving from slack to some other chat frameworks. The major reason is the loss of messages without enterprise subscription in slack. We wanted to gauge the community's feedback on the same. We are currently looking into the following chat frameworks:
1. Discord
2. Zulip
3. Rocket Chat
4. Mattermost

We welcome opinions on the above and other chat frameworks of your interest.

**Comment by jbakosi** [comment]
I would recommend matrix.org and clients: open source, federated, optionally encrypted, supports, text, images, video, audio, group, searchable history, etc.
